runtime/doc/lua.txt

```diff
@@ -1536,6 +1536,27 @@ deepcopy({orig})                                              *vim.deepcopy()*
     Return: ~
         (table) Table of copied keys and (nested) values.
 
+defaulttable({create})                                    *vim.defaulttable()*
+    Creates a table whose values are automatically created when missing.
```

Comment:
```suggestion
    Creates a table whose members are automatically created when accessed, if they don't already exist.
```

This might be useful for faking a [null-coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) style